mtest/libmscore/durationtype/tst_durationtype.cpp

```diff
@@ -71,7 +71,7 @@ void TestDurationType::halfDuration()
 
       score->startCmd();
       score->cmdAddPitch(42, false, false);
-      Chord* c = score->firstMeasure()->findChord(Fraction(0,1), 0);
+      Ms::Chord* c = score->firstMeasure()->findChord(Fraction(0,1), 0);
```

Comment:
Shouldn't the `using namespace Ms;` at the beginning of this file be enough? Why isn't the `Ms::` prefix needed anywhere else throughout the code?

Reply:
I do see https://docs.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-chord
Yet another namespace pollution from Microsoft :-(

Reply:
I'm still curious why this is the only file having this conflict, Chord surly is used in many more places, so there got to be another possible fix for this?

Reply:
there is another conflict that I found - "tst_tuplet.cpp"

Reply:
Best add to this PR, but also I'd still like an explanation why only those 2 files and not any from mscore or libmscore

Reply:
Just on a brief glance the chordview.h file in root/mscore which I randomly chose has an forward declaration of "class Chord;" inside the namespace declaration whereas these test files with issues don't, so maybe that is what is needed rather than explicitly adding Ms:: - thoughts?

Reply:
Yes, that's what I mean, better take the solution used (and successfully so) elsewhere